spec/formatting.md

```diff
@@ -695,14 +695,26 @@ These are divided into the following categories:
     > when * {Value is not one}
     > ```
     >
+    > Example **valid** messages that do _not_ result in a _Missing Selector Annotation error_:
```

Comment:
We've assigned special meaning to bold/italic text styling, so we shouldn't use them for _emphasis_.

```suggestion
    > Example valid messages that do not result in a _Missing Selector Annotation error_:
```